Library/src/main/java/BookController.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Scanner;
+public class BookController {
+    // to jest kontroler
+    public static Scanner scanner = new Scanner(System.in);
+    public static String input;
+    public static String bookToRemove;
+    public static String bookToSee;
+    public static String newTitle;
+    public static String newAuthor;
+    public static String newGenre;
+    public static String newDescrip;
+    public static String blankValue;
+    public static void letsGetStarted() {
+        BookView.greeting();
+        BookView.showMenu();
+        askAndGive();
+    }
+    public static void askAndGive() {
+        input = scanner.next();
+        BookService.doSomethingWithIt(input);
+    }
+    public static void addBook() {
+        System.out.println("What's the title?");

```

Comment:
Zamiast tego, spróbowałbym przenieść do to widoku (patrz komentarze niżej), i przekazać obiekt (nie będący Book) z informacjami z frontendu (widoku) do kontrolera. W kontrolerze możesz zmapować sobie ten obiekt na Book.

Taki pattern nazywa się DTO. Czyli miałbyś klasę BookDTO. Ja wiem, powtarza się z Book, ale zaraz tam ci dojdą informacje o tym jak składować książkę w bazie danych. A w BookDTO dostaniesz dodatkowe informacje jak z tego zrobić JSONa, YAMLa, czy inne ładnie sformatowane stringi.